pkgs/test/lib/src/runner/browser/chrome.dart

```diff
@@ -121,13 +121,16 @@ class Chrome extends Browser {
   Future<Uri> _sourceUriProvider(String sourceUrl, String scriptId) async {
     var script = _idToUrl[scriptId];
     if (script == null) return null;
+    var sourceUri = Uri.parse(sourceUrl);
+    if (sourceUri.scheme == 'file') return sourceUri;
+    // If the provided sourceUrl is relative, determine the package path.
     var uri = Uri.parse(script);
     var path = p.join(
         p.joinAll(uri.pathSegments.sublist(1, uri.pathSegments.length - 1)),
         sourceUrl);
     return path.contains('/packages/')
         ? Uri(scheme: 'package', path: path.split('/packages/').last)
-        : Uri.file(p.absolute(path));
+        : null;
```

Comment:
This doesn't seem right?

Reply:
It's a merge from **your** commit! e3bf6f6

Reply:
We shouldn't be doing merges from master in PRs we we rebase + squash them...

Reply:
uh...so...land my fixes then do the merge in another PR?

Reply:
@natebosch –  I was trying to bring in your fix

Reply:
did the merge then added my fixes on top. PTAL